operator/src/main/java/com/github/streamshub/console/api/v1alpha1/spec/security/Audit.java

```diff
@@ -0,0 +1,38 @@
+package com.github.streamshub.console.api.v1alpha1.spec.security;
+
+import java.util.Locale;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import io.fabric8.generator.annotation.Required;
+import io.sundr.builder.annotations.Buildable;
+
+@Buildable(editableEnabled = false)
+public class Audit extends Rule {
```

Comment:
Naming. Is this an AuditingRule i.e. a rule that audits its decision?

Reply:
This Audit class is used to enable audit logging for authorization decisions for resources that match the rule criteria in the parent class. Renaming to `AuditRule` would make sense.